packages/loader/driver-utils/src/treeUtils.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    SummaryType,
+    ISummaryTree,
+    SummaryObject,
+} from "@fluidframework/protocol-definitions";
+
+/**
+ * Summary tree assembler without stats collection.
+ */
+export class SummaryTreeAssembler {
```

Comment:
We already have summary tree builder (with runtime related stats) on the container runtime level. Maybe we could rename that one to `SummaryTreeBuilderWithStats`.